pkg/analytics/consts.go

```diff
@@ -0,0 +1,23 @@
+package analytics
```

Comment:
No need to define a separate file for constant, As it's not used in other packages, You can define this in `pkg/analytics/googleanalytics.go`

Reply:
Done @chandankumar4